docs/apis-clients/working-with-apis-clients.md

```diff
@@ -37,19 +26,30 @@ Clients allow applications to do the following:
 - Publish messages.
 - Update process instance variables and resolve incidents.
 
-Clients connect to Camunda Platform 8 via [gRPC](https://grpc.io), a high-performance, open source, and universal RPC protocol.
+The official clients mentioned below interact with [Zeebe](/components/zeebe/zeebe-overview.md), the workflow engine integrated into Camunda Platform 8. Clients connect to Camunda Platform 8 via [gRPC](https://grpc.io), a high-performance, open source, and universal RPC protocol.
 
 Camunda Platform 8 provides several official clients based on this API. Official clients have been developed and tested by Camunda. They also add convenience functions (e.g. thread handling for job workers) on top of the core API.
 
-### Official clients
+### Official Zeebe clients
+
+Official clients have been developed and tested by Camunda. They also add convenience functions (e.g. thread handling for job workers) on top of the core API.
 
-- [Java](java-client/index.md)
-- [Go](go-client/go-get-started.md)
-- [CLI](cli-client/index.md)
+<DocCardList items={[{type:"link", href:"/docs/apis-clients/java-client/", label: "Java client", docId:"apis-clients/java-client/index"},
+{
+type:"link", href:"/docs/apis-clients/go-client/go-get-started/", label: "Go client", docId:"apis-clients/go-client/index"
+},
+{
+type:"link", href:"/docs/apis-clients/cli-client/index", label: "CLI client", docId:"apis-clients/cli-client/index"
+}
+]}/>
+
+:::note
+Other components in Camunda Platform 8, such as [Tasklist API (GraphQL)](tasklist-api/generated.md), provide language-agnostic APIs, but no clients to interact with them. GraphQL enables you to query, claim, and complete user tasks.
+:::
 
 ### Community clients
 
-Community clients supplement the official clients. These clients have not been tested by Camunda.
+Community clients supplement the official SDKs. These clients have not been tested by Camunda.
```

Comment:
Here (but nowhere else) we talk about SDKs. Is this on purpose?

Reply:
Adjusted in commit 👍